flowauth/backend/tests/test_token_generation.py

```
@@ -89,6 +89,39 @@ def test_token_generation(
         assert approx(expiry.timestamp()) == decoded_token["exp"]
 
 
+@pytest.mark.usefixtures("test_data_with_access_rights")
+@freeze_time(datetime.datetime(year=2020, month=12, day=31))
+def test_token_list_includes_assigned_roles(
+    client, auth, app, test_user_with_roles, test_servers
+):
+    """The token list endpoint exposes the roles that were assigned at mint time."""
+    with app.app_context():
+        uid, uname, upass = test_user_with_roles
```

Ruff (0.15.12)

[warning] Unpacked variable `uid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)